bots/notify/src/main/java/org/openjdk/skara/bots/notify/issue/IssueNotifier.java

```diff
@@ -305,7 +305,18 @@ public void onNewCommits(HostedRepository repository, Repository localRepository
                             var existing = Backports.findIssue(issue, fixVersion);
                             if (existing.isEmpty()) {
                                 log.info("Creating new backport for " + issue.id() + " with fixVersion " + requestedVersion);
-                                issue = jbsBackport.createBackport(issue, requestedVersion, username.orElse(null), defaultSecurity(branch));
+                                try {
+                                    issue = jbsBackport.createBackport(issue, requestedVersion, username.orElse(null), defaultSecurity(branch));
+                                } catch (Exception exp) {
+                                    existing = Backports.findIssue(issue, fixVersion);
+                                    if (existing.isPresent()) {
+                                        log.info("Race condition occurred while creating the back port. So returning an existing backport for " + issue.id() + " and requested fixVersion "
```

Comment:
```suggestion
                                        log.info("Race condition occurred while creating backport issue, returning the existing backport for " + issue.id() + " and requested fixVersion "
```

Reply:
Sure, did the rewording as suggested, thanks.